client/src/js/samples/components/Create/__tests__/ReadSelector.test.js

```diff
@@ -0,0 +1,80 @@
+import ReadSelector from "../ReadSelector";
+
+describe("<ReadSelector />", () => {
+    let props;
+    let state;
+
+    beforeEach(() => {
+        props = {
+            files: [{ id: "foo", size: 1024, name: "bar" }],
+            error: "foo",
+            selected: ["foo", "bar"],
+            onSelect: jest.fn(),
+            handleSelect: jest.fn()
+        };
+        state = {
+            filter: "foo"
+        };
+    });
+
+    it("should render", () => {
+        const wrapper = shallow(<ReadSelector {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    it("componentDidUpdate should not call onChange() if props.file does not change ", () => {
+        const wrapper = shallow(<ReadSelector {...props} />);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wrapper' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4488032)